google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfigurationPanel.java

```diff
@@ -17,12 +17,25 @@
 package com.google.cloud.tools.intellij.appengine.cloud;
 
 import com.google.cloud.tools.intellij.resources.ProjectSelector;
+import com.google.cloud.tools.intellij.ui.BrowserOpeningHyperLinkListener;
+import com.google.cloud.tools.intellij.util.GctBundle;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Strings;
 
+import com.intellij.remoteServer.configuration.deployment.DeploymentSource;
+import com.intellij.ui.HyperlinkLabel;
+import com.intellij.ui.components.JBTextField;
+import com.intellij.util.ui.tree.TreeModelAdapter;
+
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.JCheckBox;
 import javax.swing.JLabel;
 import javax.swing.JPanel;
+import javax.swing.event.TreeModelEvent;
 
 /**
- * Common App Engine deployment configuration UI shared by flexible and standard.
+ * Common App Engine deployment configuration UI shared by flexible and standard deployments.
  */
 public class AppEngineDeploymentConfigurationPanel {
```

Comment:
there is no subclassing of it. is the google style to make everything final that can be final? what about fields, method parameters, etc.